atlascope/core/rest/endpoints/tile_endpoints.py

```diff
@@ -0,0 +1,96 @@
+from django.http.response import HttpResponse
+from django.urls import path
+from drf_yasg import openapi
+from drf_yasg.inspectors import SwaggerAutoSchema
+from drf_yasg.utils import swagger_auto_schema
+from large_image.exceptions import TileSourceError
+from large_image_source_gdal import GDALFileTileSource
+from rest_framework import mixins
+from rest_framework.exceptions import APIException
+from rest_framework.generics import GenericAPIView
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from atlascope.core.models import Dataset
+from atlascope.core.rest.additional_serializers import TileMetadataSerializer
+
+
+class TileMetadataView(GenericAPIView, mixins.RetrieveModelMixin):
+    queryset = Dataset.objects.all()
+    permission_classes = [IsAuthenticated]
+    serializer_class = TileMetadataSerializer
+
+    def get(self, *args, **kwargs):
+        dataset = self.get_object()
+        tile_source = GDALFileTileSource(dataset.source_uri)
```

Comment:
```suggestion
        tile_source = GDALFileTileSource(dataset.content)
```

When https://github.com/atlascope/atlascope/pull/25 lands.